cli/src/torii/routing.rs

```diff
@@ -528,9 +528,7 @@ impl Torii {
                 })
             });
 
-        let ws_router = events_ws_router
-            .or(blocks_ws_router)
-            .with(warp::trace::request());
+        let ws_router = events_ws_router.or(blocks_ws_router);
```

Comment:
I don't see those handlers having a `warp::trace::request()` middleware attached to them. Are they logging with some other mechanism?

Reply:
They are handled together with other endpoints at line 544.

Reply:
Oooh, I see, it was after that.